webrtc-stats.html

```diff
@@ -1836,7 +1838,10 @@
               </dt>
               <dd>
                 <p>
-                  Represents the total number of connectivity check retransmissions received.
+                  Represents the total number of connectivity check request retransmissions
```

Comment:
> responder side doesn't ever initiate retransmissions

Oh, I think you're right. I was confused about the "Resp" field in RFC7982. In that case this looks good to me.